src/utils/sort.ts

```diff
@@ -27,16 +26,16 @@ export function sortByDomain(data, property, direction = 'asc', domain) {
 
 export function sortByTime(data, property, direction = 'asc') {
   return data.sort((a, b) => {
-    const aDate = moment(a[property]);
-    const bDate = moment(b[property]);
+    const aDate = a[property].getTime();
+    const bDate = b[property].getTime();
```

Comment:
Right, but then I realized that you have to make sure **all** of the values are dates. If one value is not a date, then you have to use an ordinal scale and treat everything as strings. Some line and area charts tend to have a lot of points, so performance may become an issue.